genomics/global_platform/README.md

```diff
@@ -305,9 +205,173 @@ Allelic-variants (AVs) with maximum allele frequency <80% overlapping with codon
 | 28,394 | G     | A     | N      | R41Q   | 0.110011 | 0.3188   | 0.054947 | 0.582654 | CGG     |       2 |       3 |
 | 29,446 | G     | T     | N      | V392L  | 0.110011 | 0.770096 | 0.765545 | 0.774648 | GTG     |       2 |       2 |
 
-### Continuous analysis of pandemic data with the Galaxy API
+## Continuous analysis of pandemic data with the Galaxy API
+
+Genome surveillance projects at national levels like COG-UK produce sequencing data for unprecedented numbers of samples. To demonstrate that our system can satisfy the analysis needs of such projects, we are performing near real-time analysis of COG-UK sequencing data as it is being submitted to the European Sequence Archive (ENA). We set up an [automated analysis system](https://github.com/simonbray/ena-cog-uk-wfs) that runs our ARTIC AV-calling workflow and the reporting workflow programmatically on all new COG-UK ARTIC paired-end data via Galaxy's openly accessible API. The system also handles data organization into Galaxy histories and exports of resulting datasets. The resulting AV information is available through the [CRG Viral Beacon](https://covid19beacon.crg.eu/) project (see below for FTP access).
+
+<iframe width="100%" height="1000" frameborder="0"
+  src="https://observablehq.com/embed/@spond/sars-cov-2-cog-uk?cell=*"></iframe>
+
+### Processed COG-UK data
+
+Results of our analyses are distributed as following datasets:
+
+- BAM - deduplicated and realigned data. Only properly paired read paired with mapping quality of at least 20 are retained.
```

Comment:
Maybe instead:

are distributed as:

- publicly accessible Galaxy histories documenting the complete analysis performed for each sample and providing tabular variant report summaries for analysis batches

  URLs to these histories and the TSV datasets are listed in the JSON file [`currently_processed.json`](https://github.com/galaxyproject/SARS-CoV-2/tree/master/data/cog-uk-tracking) containing an up-to-date list of all analyzed datasets. Specifically, the tabular variant reports are part of the history listed under `report` in the JSON file and a direct link to a per-sample variant report for each batch can be found under `report` -> `datamonkey_link`.

- key result files deposited on a publically readable FTP server. These files include:
  - BAM - deduplicated and realigned data. Only properly paired read paired with mapping quality of at least 20 are retained
  - VCF - variant calls produced with `lofreq` and annotated with `SnpEff`
  - consensus FASTA - consensus sequences for each sample constructed from the called variants, in which a consensus allele is defined by an intra-sample allele frequency of >= 0.75 and in which ambiguous sites with a variant allele reaching an allele frequency between 0.25 and 0.75 are soft-masked.
  
  Use FTP client software such as [FileZilla](https://filezilla-project.org/) to access this data at `ftp://xfer13.crg.eu/`.